src/ast.mli

```diff
@@ -1,83 +1,91 @@
+type program = None
+
 type bin_op =
-    | Add    | Minus | Multiply | Divide
-    | Modulo | Equal | Neq      | Lt
-    | Leq    | Gt    | Geq      | And
-    | Or
+  Add    | Minus | Multiply | Divide
```

Comment:
So I'll remove the other leading `|` in type definitions as they're not needed. LMK if anyone objects.